index.js

```diff
@@ -20,8 +20,8 @@ function apply(options, compiler) {
     {
       // Default Glob is any directory level of scss and/or sass file,
       // under webpack's context and specificity changed via globbing patterns
-      files: arrify(options.files || defaultFilesGlob).map((file) =>
-        path.join(context, '/', file)
+      files: arrify(options.files || defaultFilesGlob).map(
+        (file) => (file.startsWith('/') ? file : path.join(context, '/', file))
```

Comment:
This will fail on windows. Should use the the absolute path utility methods available in node instead.